kiwi/xml_parse.py

```diff
@@ -730,6 +730,7 @@ class k_packagemanager_content(object):
     ZYPPER='zypper'
```

Comment:
no review needed... auto generated code

Reply:
@davidcassany would still be good if you can double check because I realized generateDS also added information for the pacman support you added with #1432 and I was confused that this data got changed. Nevertheless all auto generated changes looks good to me

Reply:
interesting, this is happening because I did not regenerate the xml_parse on my Arch Linux branch after 09e84933e. Before that commit adding a new choice element values had no impact at all on `xml_parse.py` generated code, thus xml_parse was not part of the Arch Linux branch diff. However, recent generateDS versions (we started using an updated version in 09e84933e) include the list of choices defined for elements.

The funny fact is that it never stopped working, thus I am wondering this code portion is actually being used in any way :P I kept rebasing from master on regular basis and never saw complains about not having this enum up to date. :man_shrugging:

Reply:
ok, really a funny one :)